### PR TITLE
Fixed issue in PreviewView for number textfield

### PR DESCRIPTION
**Pull request by JohanDelValleV**
Follow-up to #6901 

- [ ] Change in CHANGELOG.md described (if applicable)
- [ ] Tests created for changes (if applicable)
- [ ] Manually tested changed features in running JabRef (always required)
- [ ] Screenshots added in PR description (for UI changes)
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, submitted a pull request to the documentation repository.

### CODE REVIEW

File: src/main/resources/resource/layout/din1505/din1505winword.article.layout [hunk 1]
```diff
@@ -7,10 +7,10 @@
 \end{volume}
  (\year)
 \begin{number}
-, Nr. \number
+, Nr. {\number}
```
Comment (Siedlerchr) [inline src/main/resources/resource/layout/din1505/din1505winword.article.layout:10]: Here you don't need to add the curly braces, they are only needed when the there is a formatter (e.g HTMLChars) before.

Comment (JohanDelValleV) [reply src/main/resources/resource/layout/din1505/din1505winword.article.layout:10]: Ok, so just changing this line is supposed to work, right?
```
 "\\begin{journal}<BR><BR><i>\\format[HTMLChars]{\\journal}</i> \\end{journal} \\begin{volume}, Vol. \\volume\\end{volume}\\begin{series}<BR> \\format[HTMLChars]{\\series}\\end{series}\\begin{number}, No. \\format[HTMLChars]{\\number}\\end{number}__NEWLINE__" +
```

Comment (Siedlerchr) [reply src/main/resources/resource/layout/din1505/din1505winword.article.layout:10]: Yes! For more information: https://docs.jabref.org/collaborative-work/export/customexports

**Comment by Siedlerchr** [comment]
Thanks for finding and fixing the issue!

**Pull request merged**